### PR TITLE
Document `GET /source/{project_name}/{package_name}/_history` with OpenAPI

### PR DESCRIPTION
**Pull request by eduardoj**
Note: the parameters `delete` and `meta` will be recognized as a truthy value not only with the `?delete=1` case, but with the following cases:
- `?delete`
- `?delete=0`
- `?meta`
- `?meta=0`

### For reviewers

Access to the documented enpoint in the review-app through [this link](https://obs-reviewlab.opensuse.org/eduardoj-apidocs_newsource_package_history/apidocs-new/index.html#/Sources%20-%20Packages/get_source__project_name___package_name___history).

### CODE REVIEW

File: src/api/public/apidocs-new/paths/source_project_name_package_name_history.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
+get:
+  summary: Get package commit history.
+  description: Get package commit history.
+  security:
+    - basic_authentication: []
+  parameters:
```
Comment (krauselukas) [inline src/api/public/apidocs-new/paths/source_project_name_package_name_history.yaml:6]: @eduardoj there is also the `:view` parameter, not sure what it is for though...

Comment (krauselukas) [reply src/api/public/apidocs-new/paths/source_project_name_package_name_history.yaml:6]: https://github.com/openSUSE/open-build-service/blob/a3bd82a38dfba6ebef167057bf4da8868cab9a94/src/api/app/controllers/source_controller.rb#L315

File: src/api/public/apidocs-new/OBS-v2.10.50.yaml [hunk 2]
```diff
@@ -311,6 +311,8 @@ paths:
 
   /source/{project_name}/{package_name}:
     $ref: 'paths/source_project_name_package_name.yaml'
+  /source/{project_name}/{package_name}/_history:
```
Comment (krauselukas) [inline src/api/public/apidocs-new/OBS-v2.10.50.yaml:314]: @eduardoj sometime I'm not sure if that really needs to be documented as a seperate endpoint. In theory this path is just  `/source/{project_name}/{package_name}/{file_name}` and `_history` is the file to receive. I know its kind of a special file, but also a bit inconsistent in terms of documenting endpoints.

File: src/api/public/apidocs-new/paths/source_project_name_package_name_history.yaml (hunk 1 above)
Comment (eduardoj) [reply src/api/public/apidocs-new/paths/source_project_name_package_name_history.yaml:6]: The `:view` parameter seems not to be used in the backend, for the `/source/$project/$package/_history` path: https://github.com/openSUSE/open-build-service/blob/a3bd82a38dfba6ebef167057bf4da8868cab9a94/src/backend/bs_srcserver#L7640

It also happens with he `:expand` parameter.

File: src/api/public/apidocs-new/OBS-v2.10.50.yaml (hunk 2 above)
Comment (krauselukas) [reply src/api/public/apidocs-new/OBS-v2.10.50.yaml:314]: okay, I see, its a different endpoint on backend level https://github.com/openSUSE/open-build-service/blob/a3bd82a38dfba6ebef167057bf4da8868cab9a94/src/backend/bs_srcserver#L7640

File: src/api/public/apidocs-new/paths/source_project_name_package_name_history.yaml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+      name: deleted
+      schema:
+        type: string
+      description: Set to `1` to allow to retrieve the revision history of a deleted package.
+      example: 1
+    - in: query
+      name: limit
+      schema:
+        type: integer
+      description: Return the last `limit` elements of the revision history.
```
Comment (krauselukas) [inline src/api/public/apidocs-new/paths/source_project_name_package_name_history.yaml:19]:
```suggestion
      description: Limit the number of retrieved revision history elements to the specified number .
```

Comment (krauselukas) [reply src/api/public/apidocs-new/paths/source_project_name_package_name_history.yaml:19]: @eduardoj just a suggestion, but for me it was not really clear from the description what it does.

Comment (eduardoj) [reply src/api/public/apidocs-new/paths/source_project_name_package_name_history.yaml]: Sure. Applied.

File: src/api/public/apidocs-new/paths/source_project_name_package_name_history.yaml [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+      name: meta
+      schema:
+        type: string
+      description: Set to `1` to retrieve the revision history of the meta file (`_meta`) of the package.
+      example: 1
+    - in: query
+      name: rev
+      schema:
+        type: string
+      description: Return the revision entry specified by `rev`.
```
[the remark this answers is not in the document]
Comment (eduardoj) [reply src/api/public/apidocs-new/paths/source_project_name_package_name_history.yaml]: Also applied.

**Review by krauselukas — APPROVED** [review]

**Pull request merged**